CHANGELOG.md

```diff
@@ -1,3 +1,81 @@
+# v0.31.0 Release - 06/06/2019
+
+New Features:
+* Add CI on windows [#2214](https://github.com/GoogleContainerTools/skaffold/pull/2214)
+* Add API build trigger [#2201](https://github.com/GoogleContainerTools/skaffold/pull/2201)
```

Comment:
Maybe we should mention this is in alpha /cc @nkubala  and also link any docs on how to use it if there are any

Reply:
I think there is also a page which list all features and which stage they are in. Maybe add it there too?

Reply:
Added the alpha note to the CHANGELOG, but couldn't find any specific docs to link.

There is a doc with the deprecation policy, but since I'm not sure if the other triggers are alpha or beta, I'll add that change in another PR.

Reply:
They are totally alpha. There is no doc yet. API triggers will move around quite a bit in the next couple of weeks.